### PR TITLE
OpenTimelineIO TSC Charter

### PR DESCRIPTION
**Pull request by jminor**
This is a draft of our proposed OpenTimelineIO TSC Charter for discussion.

This PR supersedes both of these other PRs:
https://github.com/PixarAnimationStudios/OpenTimelineIO/pull/897
https://github.com/AcademySoftwareFoundation/foundation/pull/5

### CODE REVIEW

**Comment by jminor** [comment]
Note: I carefully split each intentional change from the ASWF template charter into a separate commit, so we can see the rationale behind each change.

**Comment by jminor** [comment]
@jmertic your proposed change for "template repositories" was only for inbound code. I added the same exception for outbound code as well, since we need to make the template repo available under that license as well. Does that make sense, or is that not needed?

See: https://github.com/AcademySoftwareFoundation/foundation/pull/6/commits/37c252d2b5265171048009ed2aac785ae705e5d5

**Comment by jminor** [comment]
Also, the charter lacks some clarity around the specific names & relationships between these entities:

OpenTimelineIO a Series of LF Projects, LLC
LF Projects, LLC
LFP, Inc.
Technical Advisory Council of the Academy Software Foundation of The Linux Foundation
Governing Board of the Academy Software Foundation of The Linux Foundation

mentioned these other documents, without links:
Series Agreement for the Project
Operating Agreement of LF Projects

and mentions these people/roles - but we aren't sure who those people are.
Series Manager
LF Projects Manager

**Comment by jmertic** [comment]
> @jmertic your proposed change for "template repositories" was only for inbound code. I added the same exception for outbound code as well, since we need to make the template repo available under that license as well. Does that make sense, or is that not needed?
> 
> See: [37c252d](https://github.com/AcademySoftwareFoundation/foundation/commit/37c252d2b5265171048009ed2aac785ae705e5d5)

Yes - thanks for including that. I had thought it was in a later revision but must have dropped that.

File: project_charters/opentimelineio-charter.md [hunk 1]
```diff
@@ -1,6 +1,3 @@
-<!-- SPDX-License-Identifier: CC-BY-4.0 -->
-<!-- Copyright Copyright Contributors to the [COMMUNITY NAME] Project -->
-
```
Comment (jminor) [inline project_charters/opentimelineio-charter.md]: @jmertic is this important to retain? I didn't see it in any of the other charters, but it is in the template.

Comment (jmertic) [reply project_charters/opentimelineio-charter.md]: It's fine to remove

**Comment by jmertic** [comment]
> Also, the charter lacks some clarity around the specific names & relationships between these entities:
> 
> OpenTimelineIO a Series of LF Projects, LLC
> LF Projects, LLC
> LFP, Inc.
> Technical Advisory Council of the Academy Software Foundation of The Linux Foundation
> Governing Board of the Academy Software Foundation of The Linux Foundation
> 
> mentioned these other documents, without links:
> Series Agreement for the Project
> Operating Agreement of LF Projects
> 
> and mentions these people/roles - but we aren't sure who those people are.
> Series Manager
> LF Projects Manager

Here are the answers from our legal. They also advised that this template is out of date and will be providing me with an updated template to use.

OpenTimelineIO a Series of LF Projects, LLC
We set up individual technical projects as series of one of our project hosting series LLCs.  In this case because it's software versus specification development, the project will be set up as a series of LF Projects, LLC ("LFP").  LFP operates pursuant to an Operating Agreement and the OpenTimelineIO series will be created through execution (by LFP and its sole member) of a Series Agreement for OpenTimelineIO.  You can think of these documents as the series LLC versions of a certificate of incorporation in a way.  The sole member of LFP is the Joint Development Foundation, a 501c6 nonprofit organization, and the sole member of the Joint Development Foundation is the Linux Foundation.

LF Projects, LLC
One of our series LLC entities for hosting projects.

LFP, Inc.
This name needs to be updated to the Joint Development Foundation, which merged with LFP, Inc. and the Joint Development Foundation was the surviving entity in the merger.

Technical Advisory Council of the Academy Software Foundation of The Linux Foundation
Please see the "Funding Charter" for the Academy Software Foundation ("ASWF"). This group is part of ASWF and is responsible for coordinating and facilitating communication across the projects of ASWF. 

Governing Board of the Academy Software Foundation of The Linux Foundation
Please see the "Funding Charter" referenced above. The Governing Board is responsible for allocating the budget of ASWF.

Series Agreement for the Project
See description above. 

Operating Agreement of LF Projects
See description above.

Series Manager / LF Projects Manager
These individuals are currently the same person, Michael Dolan, SVP of the Linux Foundation and Manager of LF Projects, LLC and Series Manager for all series of LF Projects, LLC.

**Comment by jmertic** [comment]
@jminor Any other open issues needing to be addressed before this is approved?

**Comment by jminor** [comment]
@jmertic you said "this template is out of date and [legal] will be providing me with an updated template to use" When can we expect that?

**Comment by jmertic** [comment]
@jminor - thanks for that note - chatting with legal they indicated that you can stick with this template. No change needed.

**Comment by jminor** [comment]
Okay great. Thanks @jmertic

**Comment by jminor** [comment]
@jmertic earlier you said "LFP, Inc. This name needs to be updated to the Joint Development Foundation, which merged with LFP, Inc. and the Joint Development Foundation was the surviving entity in the merger."
Do we need to change that in the text of the TSC Charter?

Also, does LF Projects change name also?

**Comment by jmertic** [comment]
Talking with legal, it's fine to leave both the way they are. Good thinking though @jminor

**Comment by jminor** [comment]
@jmertic I pushed an update to sections 5a, 5b, and 8 based on feedback from our legal department. Can you check to see if this looks okay?

**Comment by jmertic** [comment]
@jminor Thanks - LF legal is looking at that and will come back with their response

**Comment by jminor** [comment]
@jmertic this charter was approved by the OpenTimelineIO TSC vote on Oct 28th 2021. This PR is ready to land.

**Comment by jmertic** [comment]
Thank you @jminor !

**Pull request merged**